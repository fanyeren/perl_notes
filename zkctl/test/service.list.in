#!/bin/bash
source $(dirname $0)/common

TYPE=service
node_clear $TYPE

./zkctl -w $TYPE add --$TYPE-id=bs_dnews_1 --data={}
echo "add result=$?"
./zkctl -w $TYPE add --$TYPE-id=bs_wp_2 --data={}
echo "add result=$?"
./zkctl -w $TYPE add --$TYPE-id=bs_vip_0 --data={}
echo "add result=$?"
./zkctl -w $TYPE add --$TYPE-id=bc_group_1 --data={}
echo "add result=$?"
./zkctl -w $TYPE list | sort
echo "list result=$?"
./zkctl -w $TYPE del --$TYPE-id=bs_wp_2 
echo "del result=$?"
./zkctl -w $TYPE list | sort
echo "list result=$?"
