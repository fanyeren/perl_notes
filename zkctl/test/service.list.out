add result=0
add result=0
add result=0
add result=0
bc_group_1
bs_dnews_1
bs_vip_0
bs_wp_2
list result=0
del result=0
bc_group_1
bs_dnews_1
bs_vip_0
list result=0
